Sprint-1/exercises/decimal.js

```diff
@@ -8,8 +8,8 @@ const num = 56.5678;
 
 // Log your variables to the console to check your answers
 const wholeNumberPart = Math.floor(num);
-const decimalPart = num - wholeNumberPart ;
+const decimalPart = Math.ceil (num - wholeNumberPart) ; 
```

Comment:
This does not give the result that is expected from you. The result needs to be a decimal, but the original result gave you too many decimal places. Think about how that issue could be remedied. You should round it up, but not to a whole number.

Reply:
Hello
 thank you for reviewing my code, I did Fix it, could you check again, please?
 thank you